pkg/pilot/elasticsearch/v5/pilot.go

```diff
@@ -47,7 +47,6 @@ func NewPilot(opts *PilotOptions) (*Pilot, error) {
 		pilotInformerSynced:     pilotInformer.Informer().HasSynced,
 		esClusterLister:         esClusterInformer.Lister(),
 		esClusterInformerSynced: esClusterInformer.Informer().HasSynced,
-		localESClient:           &elastic.Client{},
```

Comment:
This was causing ```statsList, err := p.localESClient.NodesStats().NodeId(elasticsearchLocalNodeID).Do(context.Background())``` to panic, as the `localESClient.client` was nil as it wasn't constructed properly.

Instead, we now check to see if the `localESClient` is nil: https://github.com/jetstack/navigator/pull/131/files#diff-524ec275431e2337616e927829c48547R26

This has to be done because here: https://github.com/munnerz/navigator/blob/0fa9f757d2280bdf626653cbf07b610c4d37d9ae/pkg/pilot/elasticsearch/v5/pilot.go#L53

we continuously attempt to obtain an elastic client. Within the `NewClient` constructor, elastic.Client will attempt to connect to Elasticsearch and verify it's okay to do so, and returns an error if it's not, hence this 'retrying' behaviour.